imagedephi/redact/redact.py

```diff
@@ -55,11 +47,22 @@ def iter_image_files(directory: Path) -> Generator[Path, None, None]:
             yield child
 
 
+def create_redact_dir(base_output_dir: Path) -> Path:
```

Comment:
Since this is a simple clean function, it might be nice to have a short docstring explaining its black-box behavior.

Reply:
added in a0c2bf04704b088fb3ba1cfdeb17ff930b8983bf